static/js/components/PersonalForm.js

```diff
@@ -39,6 +39,37 @@ export default class PersonalForm extends ProfileFormFields {
     ui:                     UIState,
   };
 
+  renderRomanizedFields = () => {
+    const { profile } =  this.props;
+    const english = /^[A-Za-z0-9]*$/;
```

Comment:
I think we want to allow characters that fit in CP-1252 beyond the alphanumeric range so that we can accept most European names without the need for the user to convert. That's roughly the ranges `0x0020` - `0x007F` and `0x00A0` - `0x00FF` in Unicode (the gap at the begging and in the middle excludes control characters and some characters that map oddly into unicode). There are also a few non-punctuation characters in the 128-159 range of CP-1252 (the middle gap) that might be worth including as well.

For more details: https://en.wikipedia.org/wiki/Windows-1252#Code_page_layout